src/client/src/components/pages/errors/index.jsx

```diff
@@ -5,12 +5,11 @@ export default function index(props) {
   const { status } = props;
   return (
     <div className="error">
-      <h1>{status}</h1>
+      <h1>{status} </h1>
       <p>Oops! Something is wrong.</p>
       <a className="button" href="/">
-        <i className="icon-home" /> Go back in initial page, is better.
+        <i className="icon-home" /><i class="fas fa-undo-alt"></i>  Go back in initial page, is better.
```

Comment:
remove is better
for UX :smile:

Reply:
@mohannadhanafi :wink: 
this page is added by you :smile: 
I was just edit style :ok_hand: